### DISCUSSION TITLE
Is there a way to change the '- State -' string in the selectorManager dropdown?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5479] [Q&A] [answered]
In my case I would rather name it "Default", but I can't find a way to change it.

on the init configuration I can set an array of states and give each one a name; but those are the 'extra' states to the default state.

I thought about doing it via I18n too, but I don't really know how.

Thanks!

**Comment by javadsamiee** [comment, marked as answer]
Maybe it will help you:

```
      const I18nManager = editor.I18n;
      I18nManager.addMessages({
        en: { selectorManager: { emptyState: 'New state' } },
      });
```
